### ISSUE TITLE
Add newer research links to the "Research" page

### ISSUE THREAD

**Issue by jaclinec** [uswds/uswds-site#2819]
We link to some key research studies from the USWDS Research page. We should add some updated research projects there since we've recently completed and them made public.

- Disabled states literature review - late 2023
- "Zebra" batch component testing - spring 2024

So, under "Key research and research operations" the list should read:

- ["Zebra" component usability testing](https://github.com/uswds/uswds/wiki/%E2%80%9CZebra-batch%E2%80%9D-component-usability-research-findings-Q1-2024) (spring 2024): Testing another batch of components with participants with various disabilities.
- ["Buffalo" component usability testing](https://github.com/uswds/uswds/wiki/Usability-research-findings-for-Q3-2023) (summer 2023): Testing components with visually impaired participants. We also discussed this research at the [October 2023 USWDS monthly call](https://youtu.be/_uNXQpu6Dwg?feature=shared).
- [Usability testing day checklist](https://github.com/uswds/uswds/wiki/Usability-testing-day-checklist): Steps to follow on the day of testing. This checklist also prepares researchers to account for assistive technology like screen readers.
- [Disabled states literature review](https://github.com/uswds/uswds/wiki/Disabled-States-Research-Findings-2023) (summer 2023): Finding out whether disabled states should be used and how.
- [Top Tasks](https://digital.gov/event/2023/04/20/uswds-monthly-call-april-2023/) (spring 2023): Finding out what matters most to USWDS users.
- [Inclusive Design Patterns research](https://designsystem.digital.gov/together/) (fall 2022): Reducing barriers to participation in common government interactions.
- [Transforming the American digital experience](https://designsystem.digital.gov/next/) (spring 2021): Supporting emergency response efforts related to the COVID-19 and identifying strategies for advancing the design system through agency collaboration.

I also changed the month names to seasons (i.e. summer, fall, etc.) to make them more scannable.

**Comment by finekatie** [comment]
Hi @jaclinec , will this replace the current list completely? Just checking. I know some of the items are the same. :)

**Comment by jaclinec** [comment]
@finekatie yes I just wanted to list is all here to make it easy, since some little things (like changing from month names to seasons for example) are changing for many of them. So short answer is yes, this should replace the current list.
